docs/api-spec.md

```diff
@@ -42,288 +39,100 @@ Before composing a track, you'll need to create/initialize a track.
 On successful creation, you'll get a track object in response.
 
 **Example**
-
 ```json
 {
-  "status": "Created successfully",
-  "track": {...},
-  "uuid": "81b38b35-6535-4230-9e34-7c123e7d1348"
+   "status": "created",
+   "project_id": "3ade3151-372d-4ac8-b1ef-866a67ef0875",
```

Comment:
this is updated now, I am not sending project_id and total_options since its not required by the API user